mailpoet/assets/js/src/notices/notice.tsx

```diff
@@ -36,7 +36,7 @@ function Notice({
     if (timeout) {
       timeoutRef.current = setTimeout(close, timeout);
     }
-    return () => (timeoutRef.current ? clearTimeout(timeoutRef.current) : null);
+    return () => (timeoutRef.current ? clearTimeout(timeoutRef.current as NodeJS.Timeout) : null);
```

Comment:
Two things here:
1. `NodeJS.Timeout` will not be the correct type as this is run in browser where it doesn't exist. You can simply say "whatever `setTimeout` returns" - `ReturnType<typeof setTimeout>`.
2. You should avoid type assertions (`as`) as much as possible and type the original data, so `const timeoutRef = React.useRef<ReturnType<typeof setTimeout>>(null)` here.

Reply:
Addressed in 7ca118a5364857f3ab95a64ba4783e3b82388716